Src/Newtonsoft.Json-for-Unity/README.md

```diff
@@ -4,6 +4,22 @@ Json.NET is a popular high-performance JSON framework for .NET
 
 This package is a fork of Newtonsoft.Json containing custom builds targeting standalone, portable (UWP, WP8), and AOT targets such as all IL2CPP builds (iOS, WebGL, Android, Windows, Mac OS X, et.al).
 
+## Versioning format
+
+_Staying with JamesNK's version syntax, but with a twist :dizzy:_
+
+Based off JamesNK's versioning, but with the addition of two digits on the last segment.
+This is for Newtonsoft.Json-for-Unity to be able to have independent releases,
+at the same time still being easy to see which version of Json.NET it's based of.
+
+![explanation of version][version-explanation.png]
+
+Where official Json.NET 12.0.1 becomes Newtonsoft.Json-for-Unity 12.0.1*xx*.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[emphasis-marker] Emphasis should use `_` as a marker](https://app.codacy.com/app/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4554595)